18F/identity-idp
# LG-11012: redo document capture bug fixes

## 🎫 Ticket
[LG-11012](https://cm-jira.usa.gov/browse/LG-11012)

## 🛠 Summary of changes
- reset redo_document_capture in idv_session upon successful doc auth in the hybrid flow -- same as in the standard flow
- allow a user to perform hybrid document capture during a document capture redo
- doc capture polling to return new doc auth result (not original result) during a document capture redo
- confirm ocr_pending_confirmation after a user confirms/submits  a succesful doc auth
- add a timestamp to the DocumentCaptureSessionResult

## 📜 Testing Plan
Scenario1: Standard doc auth + hybrid redo doc auth
1. Complete IdV until hybrid handoff is reached
2. Select desktop and submit images  (that trigger barcode attention) and continue
3. continue IdV until verify info page is reached
4. On verify info page, click the "upload new photos" link in the warning banner at top of page
5. Select hybrid/send link and submit images successfully
6. return  to original window and complete IdV
 
Scenario2: hybrid doc auth + hybrid redo doc auth
1. Complete IdV until hybrid handoff is reached
2. Select hybrid/sendlink and fetch link at example.com/test/telephony
3. Open mobile link in incognito/private window and submit images  (that trigger barcode attention) and continue
4. return  to original window and continue IdV until verify info page is reached
5. On verify info page, click the "upload new photos" link in the warning banner at top of page
6. Select hybrid/send link and fetch link at example.com/test/telephony
7. Open mobile link in incognito/private window and submit images  (that trigger barcode attention) and continue
8. return  to original window and complete IdV
